src/components/UserUpdateAvatarForm.js

```diff
@@ -0,0 +1,45 @@
+import Popup from "./Popup.js";
+
+export default class UserUpdateAvatarForm extends Popup {
```

Comment:
`UserUpdateAvatarForm` is not needed because you should create the `avatar` modal using `PopupWithForm`, to avoid logic duplication